### PR TITLE
Added runAs Variables for deployments with restricted PSS

### PR DESCRIPTION
**Pull request by cd-at-iteratec**

### CODE REVIEW

File: charts/dvpe-deployment-gloo/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
 
+## [4.5.0]
```
Comment (mithie) [inline charts/dvpe-deployment-gloo/CHANGELOG.md:9]: 4.6.0

File: charts/dvpe-deployment-gloo/CHANGELOG.md
Comment (mithie) [inline charts/dvpe-deployment-gloo/CHANGELOG.md:1]: please add 4.6.0 link target at bottom of page

File: charts/dvpe-deployment-gloo/templates/deployment.yaml [hunk 2]
```diff
@@ -41,6 +41,9 @@ spec:
           {{- toYaml .Values.deployment.podAnnotations | nindent 8 }}
         {{- end }}
     spec:
+      securityContext:
+        runAsUser: {{ int .Values.deployment.spec.userId }}
```
Comment (mithie) [inline charts/dvpe-deployment-gloo/templates/deployment.yaml:45]:
```suggestion
        runAsUser: {{ int .Values.deployment.spec.securityContext.userId }}
```

File: charts/dvpe-deployment-gloo/templates/deployment.yaml [hunk 3]
```diff
@@ -41,6 +41,9 @@ spec:
...
+        runAsGroup: {{ int .Values.deployment.spec.groupId }}
```
Comment (mithie) [inline charts/dvpe-deployment-gloo/templates/deployment.yaml:46]:
```suggestion
        runAsGroup: {{ int .Values.deployment.spec.securityContext.groupId }}
```

File: charts/dvpe-deployment-gloo/values.yaml [hunk 4]
```diff
@@ -23,6 +23,10 @@ deployment:
   # deployment.podAnnotations -- Object of additional `podAnnotations`.
   podAnnotations: {}
   spec:
```
Comment (mithie) [inline charts/dvpe-deployment-gloo/values.yaml:25]:
```suggestion
  spec:
    securityContext:
      userId: ...
      groupId: ...
```

**Review by mithie — CHANGES_REQUESTED** [review]

File: charts/dvpe-deployment-gloo/templates/deployment.yaml (hunk 3 above)
Comment (cd-at-iteratec) [reply charts/dvpe-deployment-gloo/templates/deployment.yaml:46]: done

File: charts/dvpe-deployment-gloo/values.yaml (hunk 4 above)
Comment (cd-at-iteratec) [reply charts/dvpe-deployment-gloo/values.yaml:25]: done

File: charts/dvpe-deployment-gloo/templates/deployment.yaml (hunk 2 above)
Comment (cd-at-iteratec) [reply charts/dvpe-deployment-gloo/templates/deployment.yaml]: done

**Review by mithie — APPROVED** [review]

**Pull request merged**